modules/nextflow/src/main/groovy/nextflow/processor/TaskRun.groovy

```diff
@@ -303,6 +303,11 @@ class TaskRun implements Cloneable {
      */
     volatile int failCount
 
+    /**
+     * The number of times the submit of the task has been retried
+     */
+    volatile int submitRetry
```

Comment:
Perhaps it should be `submitAttempt` to be consistent with `task.attempt`

Reply:
makes sense